sonar-db/src/main/resources/org/sonar/db/component/ComponentMapper.xml

```diff
@@ -65,12 +65,12 @@
   </select>
 
   <select id="selectByProjectUuid" parameterType="string" resultType="Component">
-    SELECT
+    select
     <include refid="componentColumns"/>
-    FROM projects p
-    <where>
-      p.project_uuid=#{projectUuid}
-    </where>
+    from projects root
+    inner join projects p on p.project_uuid=root.uuid and p.organization_uuid=root.organization_uuid
```

Comment:
As discussed SQL inner joins should be dropped. I suppose it will be done in another PR ?